projects/react-boilerplate-ts/app/global-styles.ts

```diff
@@ -1,6 +1,6 @@
 import { injectGlobal } from 'styled-components';
 
-/* eslint no-unused-expressions: 0 */
+/* tslint:disable */
```

Comment:
is there no equivalent to `no-unused-expressions` in tslint? better if there is a specific rule disabled as it shows the thinking behind why it's there.